tests/main/interfaces-home/task.yaml

```diff
@@ -0,0 +1,144 @@
+summary: Ensure that the home interface works.
+
+details: |
+  The home interface allows a snap to access non-hidden files in $HOME
+
+  A snap which defines the home plug must be shown in the interfaces list.
+  The plug must be autoconnected on install and, as usual, must be able to be
+  reconnected.
+
+environment:
+  SNAP_NAME: home-consumer
+  SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+  READABLE_FILE: "$HOME/readable"
+  HIDDEN_READABLE_FILE: "$HOME/.readable"
+  WRITABLE_FILE: "$HOME/writable"
+  PREVIOUS: "previous content"
+  CREATABLE_FILE: "$HOME/creatable"
+  HIDDEN_CREATABLE_FILE: "$HOME/.creatable"
+  OK: "ok"
```

Comment:
I'm merging this to not delay it further, but I really meant using "ok", the string, instead of a variable, and "previous" the string as well. The variables here add noise without benefit.